symbols/arm9.yml

```diff
@@ -29,16 +29,19 @@ arm9:
   subregions:
     - itcm.yml
   functions:
-    - name: SvcWaitByLoop
-      address:
-        NA: 0x2000088
-      description: Software interrupt.
     - name: SvcSoftReset
       address:
+        EU: 0x20001A4
```

Comment:
Can you audit that these addresses are correct? While it's gratifying that my sorting algorithm's conflict resolution procedure is coming into play, it makes me suspicious that we have a genuine instance where the symbol order differs across versions. Especially in the system functions at the start, where I would expect things to be virtually identical.

Reply:
Yes, these are correct. All of these interrupt functions are mixed around between US and EU for some reason, and there are calls to them within the regular ARM9 code that resolve correctly for both versions with these specific addresses.

Reply:
Weird